### DISCUSSION TITLE
Issues with Remotely on Nginx proxy manager please help

### DISCUSSION THREAD

**Discussion by Baltimorepc** [immense/Remotely#744] [Q&A] [answered]
i followed the video and also use nginx proxy manager but when i am in the subdomain i cant save anything or open the ? marks, i can click on the left side and select options but once in those option i cant do anything on the right side as in in the option i selected on the left, but if i go to the manual ip it works fine there. any ideas would be grateful.

**Comment by Baltimorepc** [comment]
ive been trying to fix this for almost a week. none of the buttons works when on the subdomain i know this is a nginx issues since the machine isnt the same as my npm but i also added the npm IP to the proxy list as mentioned in gitub but that also didnt work can anyone help i am stuck

**Comment by lukaspavelka** [comment]
Is there any manual how to proper configure Nginx proxy manager ? I also need help with it....

**Reply by Baltimorepc** [reply]
I wish i have looked and have found multiple ways to fix it and each way leads to the same thing and same issue. i guess its what i get for running things from separate machines. as my website and everything else works fine if i run it all form the same machine but i want to use my test server not my actual server

i can help u as much as i can if you have it all on one but if u have it like mine i can only get you to where i am :) but i would be happy to help

**Reply by Baltimorepc** [reply]
i tested today and even though of these issues the download page and remote page all work and was able to do a remote support with no issues. just set it up thru internal ip:port  at least for a work around in till some one can assist us

**Reply by Baltimorepc** [reply, marked as answer]
i was able to resolve this issue. my issue was with my cloudflare. let me know and i can try and assist you

**Reply by lukaspavelka** [reply]
Looks like after docker update to last version all start working well. Thanks for help

**Reply by mendesit** [reply]
@Baltimorepc could you please put the solution here?

Remotely doesn't work trough cloudflare zero trust. Can you helpe me, please? :D

**Reply by Baltimorepc** [reply]
i can do my best, have you double checked your cloud flare settings and make sure ur not Minifying anything
